piserver/app.py

```diff
@@ -52,14 +52,14 @@ def handle_schedule():
     if request.method == 'DELETE':
         return smart_blinds_system.deleteSchedule()
 
-@app.route( COMMAND_ROUTE, methods=[ 'POST' ] )
-def post_command():
-    # TODO parse command, likely extract to new function once we are more clear 
-    # on the schema
-    position = request.form[ 'position' ]
-    duration = request.form[ 'time' ]
+@app.route( COMMAND_ROUTE, methods=[ 'POST', 'DELETE' ] )
+def handle_command():
+    if request.method == 'POST':
+        command = request.json
+        return smart_blinds_system.postBlindsCommand( command )
```

Comment:
postBlindsCommand takes in a dict, but then you pass it a json? Is that okay?

Reply:
`request.json` will convert the request data, provided it is a json, into a dict or something equivalent that can be accessed in the same way. I tested this with curl on localhost and it behaves as expected.

Reply:
okay, sounds good